dags/openshift_nightlies/manifest.yaml

```diff
@@ -142,6 +142,11 @@ platforms:
         config:
           install: rosa/rosa-hcp-ovn.json
           benchmarks: hosted-control-plane-p75.json
+      - name: rosa-hcp-data-plane
+        schedule:  "1 12 * * 3"
+        config:
+          install: rosa/rosa-hcp-ovn.json
+          benchmarks: data-plane-v2.json
```

Comment:
I think we should have different installation configs for control-plane and data-plane benchmarks (similar to self-managed ones)

In the case of control-plane I'd go with r5.xlarge (4x32) infra nodes since is used in a control-plane test with just 9 worker nodes

Reply:
I will add one.